gemrb/GUIScripts/InventoryCommon.py

```diff
@@ -242,6 +242,23 @@ def GetItemDescription (item, itemtype):
 	if (itemtype & 2):
 		text = item["ItemDesc"]
 
+	if GameCheck.IsIWD2():
+		# restrictions apply, we only list them
+		usabilityMask = item["UsabilityBitmask"]
+		if usabilityMask > 0:
+			text = GemRB.GetString(text) + "\n\n"
+			text += f"[color=ffff00ff]{GemRB.GetString(31530)}[/color]\n  "
```

Comment:
f-strings were introduced in python 3.6, while we still advertise 3.3 as the minimum, so please use .format() instead.

Reply:
Wow, I didn't realize how old Python 3 already is ...

Reply:
Why the revert?

Reply:
I messed up with git ...